draft-packages/loading-spinner/KaizenDraft/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -77,3 +78,11 @@ export const LoadingSpinner = ({
     )}
   </div>
 )
+
+/**
+ * @deprecated draft-packages LoadingSpinner is deprecated. Please use Button from "@kaizen/loading-spinner" instead.
```

Comment:
It still says Button 😆 

```suggestion
 * @deprecated draft-packages LoadingSpinner is deprecated. Please use LoadingSpinner from "@kaizen/loading-spinner" instead.
```

Reply:
I could have sworn I commited this and pushed it up... Must have been a fever dream 😢